### ISSUE TITLE
MySQL compatibility: SET net_write_timeout

### ISSUE THREAD

**Issue by slvrtrn** [ClickHouse/ClickHouse#58553]
### Description
Required for AWS QuickSight SPICE import.

Sample query:

```sql
SET net_write_timeout = 60
```

### Expected behavior

Either ignored completely or translated to [send_timeout](https://clickhouse.com/docs/en/operations/settings/settings#connect-timeout-receive-timeout-send-timeout) (?)

See also: https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_net_write_timeout

**Comment by alexey-milovidov** [comment]
Yes, let's make it translated to `send_timeout`, and similarly, `net_read_timeout` to `receive_timeout`.

**Comment by rschu1ze** [comment]
@slvrtrn `net_write_timeout` and `net_read_timeout` are MySQl-proprietary settings and only meaningful in MySQL contexts. This is different from other MySQL SQL extensions (e.g. function syntax, extended type conversions, etc.) which are also MySQL-proprietary but also useful within ClickHouse. For that reason, we should implement `net_write_timeout` and `net_read_timeout` rather outside the ClickHouse core.

I like option 1. Going from four to five replacements is not exactly "crowded" :-) Also, the replacement rules don't overlap, so I don't see a danger of messing something up during replacement.

Option 2: Maybe later, option 1 seems better.

Option 3: We don't, I guess this is because 99% of all settings are ClickHouse-specific and we don't need compatibility aliases with other databases.

Option 4: Let's do option 1!
